src/abc_atlas_access/abc_atlas_cache/manifest.py

```diff
@@ -0,0 +1,284 @@
+from typing import Dict, List, Any, Union, Tuple, Optional
+import json
+import pathlib
+from abc_atlas_access.abc_atlas_cache.file_attributes import \
+    CacheFileAttributes  # noqa: E501
+
+"""Methods for accessing and manipulating manifest.json files associated with
+the ABC atlas data. Adapted from
+https://github.com/AllenInstitute/AllenSDK/blob/master/allensdk/api/cloud_cache/manifest.py  # noqa: E501
+"""
+
+
+class Manifest(object):
```

Comment:
Given that you've already spent time internalizing the design of the `manifest.json`, how would you feel about adding a `.md` file somewhere in here (presumably under `src/abc_atlas_acces/docs` so that it does not interfere with the user-facing documentation) explaining what you have learned about the design of the `manifest.json` for future developers?

Feel free to push back on this.

Reply:
Sure I can start writing something up. It's not too interesting so far. Will be more important once I start putting the cache together.

Reply:
Thanks.